cli/config.go

```diff
@@ -75,6 +75,7 @@ func defaultConfig() *viper.Viper {
 
 	cfg.SetDefault("datastore.badger.path", "data")
 	cfg.SetDefault("datastore.encryptionDisabled", false)
+	cfg.SetDefault("keyring.service", "defradb")
```

Comment:
suggestion: "namespace" might be a more approachable/understandable name from an external UX POV